components/server/src/grpc/object.rs

```diff
@@ -351,9 +351,12 @@ impl ObjectService for ObjectServiceImpl {
         let request = CloneObject(request.into_inner());
         let object_id = tonic_invalid!(request.get_object_id(), "Invalid object id");
         let (parent_id, parent_mapping) = tonic_invalid!(request.get_parent(), "Invalid object id");
-        let ctx = Context::res_ctx(parent_id, DbPermissionLevel::WRITE, true);
+        let parent_ctx = Context::res_ctx(parent_id, DbPermissionLevel::APPEND, true);
```

Comment:
This is wrong, both parent_ctx and object_ctx use the same id (`parent_id`)

Reply:
Ohhh, typo!